src/zope/app/apidoc/typemodule/type.py

```diff
@@ -69,10 +69,11 @@ def items(self):
         """See zope.container.interfaces.IReadContainer"""
         results = [(name, LocationProxy(iface, self, name))
                    for name, iface in getUtilitiesFor(self.interface)]
-        results.sort(lambda x, y: cmp(x[1].getName(), y[1].getName()))
+        results.sort(key=lambda x: x[1].__name__)
```

Comment:
The thing is, these usually (or always, at least in tests) *are* interfaces. The only interfaces that are registered are `InterfaceClass zope.browsermenu.interfaces.IMenuItemType` and `InterfaceClass zope.publisher.interfaces.browser.IBrowserSkinType`. And the "utilities" that are registered are actually interfaces like `InterfaceClass zope.browsermenu.interfaces.AddMenu`, `InterfaceClass zope.app.menus.zmi_actions` and `InterfaceClass zope.app.rotterdam.Rotterdam`. 

I think the test setup is probably just wrong.